file-formats/aifd/examples/z_aff_example_aifd.aifd.json

```diff
@@ -0,0 +1,19 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example AIFD for ABAP File Format",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "aifInterfaces":[
+    {
+      "interfaceObjectName":"INT_OBJ_1"
```

Comment:
@D047539 did you consider to change `interfaceObjectName` to `name`? Typically, the array name, here 'aifInterfaces', is sufficient and in particular if the array element contains a single String.

Reply:
@albertmink no, we have not changed the name from interfaceObjectName to name because name is very generic one and from our point of view it is not a clear name. We would like to keep the interfaceObjectName as name.